src/python/PyImath.pc.in

```diff
@@ -12,5 +12,5 @@ libsuffix=@LIB_SUFFIX_DASH@
 Name: PyImath
 Description: Python bindings for the Imath libraries
 Version: @IMATH_VERSION@
-Libs: -L${libdir} -lImath${libsuffix} -lPyImath${libsuffix}
+Libs: -L${libdir} -lImath${libsuffix} -lPyImath@PYIMATH_LIB_PYTHONVER_ROOT@@Python3_VERSION_MAJOR@_@Python3_VERSION_MINOR@${libsuffix}
```

Comment:
This all looks good to me. But shouldn't the reference to "Python3" be agnostic regarding 3 vs 2? Will this work for python2?

Reply:
Good catch. I doubt this works with python2, although I haven't tested it.

Can I safely assume, that Imath only supports one python version at any time, i.e. no parallel installations of e.g. libPyImath_Python2_7-3_0.so and libPyImath_Python3_8-3_0.so?

If so, we could add a variable, something like PYIMATH_PYTHON_SUFFIX in the blocks at https://github.com/AcademySoftwareFoundation/Imath/blob/master/src/python/CMakeLists.txt#L106 and line 114, by using the `PYIMATH_LIB_PYTHONVER_ROOT` and corresponding `Python{2,3}_VERSION_{MAJOR,MINOR}` vars and substitue this one variable in PyImath.pc.in. What do you think?